src/lib/simulation/tob.ts

```diff
@@ -98,16 +98,19 @@ class TheatreOfBloodClass {
 			loot.add(NonUniqueTable.roll());
 		}
 
+		let clueRate = 3 / 25;
 		if (isHardMode) {
 			// Add 15% extra regular loot for hard mode:
 			for (const [itemID] of Object.entries(loot.bank)) {
 				loot.bank[Number.parseInt(itemID)] = Math.ceil(loot.bank[Number.parseInt(itemID)] * 1.15);
 			}
 			// Add HM Tertiary drops: dust / kits
 			loot.add(HardModeExtraTable.roll());
+
+			clueRate = 3.5 / 25;
 		}
 
-		if (roll(25)) {
+		if (Math.random() < clueRate) {
```

Comment:
You could just do `if (roll(clueRate)) {` here

Reply:
roll() returns true or false so a comparison to a number wouldn't work. You could convert 3/25 and 3.5/25 to percentages and use percentChance()

Reply:
also correct me if I'm wrong but doesn't Math.random() return a float between 0 and 1? so this would always be true.